src/app/containers/article-info-form-dialog/index.tsx

```diff
@@ -43,28 +83,81 @@ export const ArticleInfoFormDialog: React.FC<{}> = () => {
 
   return (
     <section className={classes.root}>
-      <TextField
-        fullWidth
-        name="articleDOI"
-        label="Article DOI"
-        classes={{ root: classes.inputField }}
-        InputLabelProps={labelProps}
-        variant="outlined"
-        multiline
-        value={userArticleInfo.articleDOI}
-        onChange={handleFormChange}
-      />
-      <TextField
-        fullWidth
-        name="publisherId"
-        label="Publisher ID"
-        classes={{ root: classes.inputField }}
-        InputLabelProps={labelProps}
-        variant="outlined"
-        multiline
-        value={userArticleInfo.publisherId}
-        onChange={handleFormChange}
-      />
+      <div className={formGrid.container}>
+        <Select
+          className={formGrid.fullWidth}
+          name="subjects.0"
+          placeholder="Please select"
+          fullWidth
+          blankValue={undefined}
+          label="Subject *"
+          value={userArticleInfo.subjects[0]}
+          onChange={handleSubjectChange}
+          options={SubjectOptions}
+        />
+        <Select
+          className={formGrid.fullWidth}
+          name="subjects.1"
+          placeholder="Please select"
+          fullWidth
+          blankValue={undefined}
+          label="Subject"
+          value={userArticleInfo.subjects[1]}
+          onChange={handleSubjectChange}
+          options={SubjectOptions}
+        />
+        <TextField
```

Comment:
No control to edit the article type, but I can't recall if we need it for MVP or not. (The design has it, but also has lots of other bits that we aren't bothering with)

Reply:
We don't. I think it's always defined in XML and not changed in editor. Not for MVP at least